server/api/dbConfig/storyDatabase.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS storyDatabase;
```

Comment:
If you drop the storyDatabase table, the existing data would be gone after your server is restarted.